src/templates/blog-post.jsx

```diff
@@ -97,3 +90,19 @@ export const query = graphql`
 `;
 
 export default BlogPostTemplate;
+
+export const Head = ({
+  data: {
+    mdx: { frontmatter },
+  },
+}) => (
+  <SEO
+    title={SEO_DATA.blogPost({ title: frontmatter.title }).title}
+    description={SEO_DATA.blogPost({ description: frontmatter.summary }).description}
+    ogImage={
+      SEO_DATA.blogPost({
+        ogImage: frontmatter.ogImage.childImageSharp.gatsbyImageData.images.fallback.src,
+      }).ogImage
+    }
+  />
+);
```

Comment:
After destructuring, you can pass it directly to SEO component.
```suggestion
}) => {
const ogImageUrl = getSrc(ogImage); // import getSrc from gatsby-plugin-image, check for more https://www.gatsbyjs.com/docs/reference/built-in-components/gatsby-plugin-image/#getsrc
return (
  <SEO {...SEO_DATA.blogPost({title, description: summary, ogImage: ogImageUrl})} />
);
```